packages/orbit-components/src/Modal/ModalFooter/index.tsx

```diff
@@ -16,10 +16,6 @@ const StyledChild = styled.div<{ flex?: Props["flex"] }>`
     flex: ${flex};
     box-sizing: border-box;
     padding: ${rtlSpacing(`0 ${theme.orbit.spaceXSmall} 0 0`)};
-
-    ${media.largeMobile(css`
```

Comment:
I guess it's better of just removal, add there `width: 100% and display:flex;`

Reply:
I would refrain from adding display properties to inner wrapper elements that can't be controlled. If devs need a display flex, they'll add it with a wrapper component. But if they don't want it, it will always be there… What would be the advantage of adding a display and a width there?

Reply:
Ok, I forgot, that flex prop still be applied there, so no need for that then